doc/files/package.json.md

```diff
@@ -108,8 +107,34 @@ You can check [the full list of SPDX license IDs](https://spdx.org/licenses/).
 Ideally you should pick one that is
 [OSI](http://opensource.org/licenses/alphabetical) approved.
 
-It's also a good idea to include a LICENSE file at the top level in
-your package.
+If your package is licensed under multiple common licenses, use an SPDX license
+expression syntax version 2.0 string, like this:
+
+    { "license" : "(ISC OR GPL-3.0)" }
+
+If you are using a license that hasn't been assigned an SPDX identifier, or if
+you are using an uncommon or custom license, do not include a "license" string
+in package.json. In those cases especially, but also more generally, it's a good
+idea to include a LICENSE file at the top level of the package.
```

Comment:
Even if `npm` checked for `LICENSE`, the guidelines recommend that all packages have a `LICENSE` file. It won't be clear when a `LICENSE` exists whether `package.json` should have `license` or not, since `LICENSE` could contain a standard or non-standard license.

Though [GitHub](https://developer.github.com/v3/licenses/#get-a-repositorys-license) and others are trying it, detection of standard licenses in plaintext is non-trivial. I may go there someday, but I doubt `npm` really wants to, or should.